### PR TITLE
Route53 Healthchecks

### PR DESCRIPTION
**Pull request by emileswarts**
This introduces the Aws SDK for Route53.
The process of establishing whether an IP is healthy requires 2 steps:
  - Get healthcheck Id for that IP, excluding the latency health check
  - Do another API request to establish whether the health check is
  healthy based on the 6 health checkers associated with it

This doesn't display it on the view yet.
There is a failing feature spec for this that isn't a part of this PR.

This creates the use cases and wraps tests around them with fake AWS
gateways.

### CODE REVIEW

**Comment by kjdchapman** [comment]
At a scan, I like this, but I'm finding the use case itself a little hard to parse. 

I'd like to check it out and play with a little bit to get my head round it, but I'm not sure if I'll get to that today.

I don't have any major complaints if it'd block you to wait until I can get to it (I'm certainly happy if someone else reads and approves).

**Comment by tommotaylor** [comment]
LGTM but like Kyle this is very new code and domain to me.
